src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,64 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import cn from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { ResponseError } from '../../types/ResponseError';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  addMovies: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovies }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [loadError, setLoadError] = useState<ResponseError | null>(null);
+  const [isDataLoading, setIsDataLoading] = useState(false);
+
+  const loadMovie = async (
+    currentQuery: string,
+    event:FormEvent<HTMLFormElement>,
+  ) => {
+    event.preventDefault();
+
+    setIsDataLoading(true);
+
+    try {
+      const data = await getMovie(currentQuery);
+
+      if ('Error' in data) {
+        setLoadError(data);
+        setIsDataLoading(false);
+      } else {
+        setMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster === 'N/A'
+            ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+            : data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}`,
+          imdbId: data.imdbID,
+        });
+        setIsDataLoading(false);
+      }
+    } catch {
+      setLoadError({
+        Response: 'False',
+        Error: 'unexpected error',
+      });
+      setIsDataLoading(false);
+    } finally {
+      setIsDataLoading(false);
+    }
```

Comment:
You don't need to do it twice. Choose one.

Reply:
agree, finally will set loader to false always